workloads/services/k8s/kubeflow-install/kustomize/kubeflow-apps/kustomization.yaml

```diff
@@ -1,18 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-
-# TODO: Remove this when the bug is fixed in v1.3 # BUG: https://github.com/kubeflow/manifests/pull/1686/files
-images:
```

Comment:
Did you notice any easy way to default NGC containers and start Jupyter by default in the new version?